### PR TITLE
fix(app): #154 Fixed Window Size when restoring window

### PR DESCRIPTION
**Pull request by sshcrack**
Added a _PrevSize Variable to Program in order to restore the previous size

### CODE REVIEW

File: RePlays.csproj [hunk 1]
```diff
@@ -51,13 +52,22 @@
   <PropertyGroup>
     <NugetTools>$(PkgNuGet_CommandLine)\tools</NugetTools>
     <SquirrelTools>$(PkgClowd_Squirrel)\tools</SquirrelTools>
-    <Version>1.0.0</Version>
+    <Version>1.3.1</Version>
```
Comment (lulzsun) [inline RePlays.csproj:55]: I don't think it is necessary to change the version here. Version is handled during the `dotnet publish` where we specifically pass in a version number.

File: RePlays.sln [hunk 2]
```diff
@@ -1,20 +1,23 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31911.196
+# Visual Studio Version 17
+VisualStudioVersion = 17.9.34728.123
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RePlays", "RePlays.csproj", "{10BA385C-0777-4293-BED2-C85223FC2CF8}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RePlays", "RePlays.csproj", "{10BA385C-0777-4293-BED2-C85223FC2CF8}"
```
Comment (lulzsun) [inline RePlays.sln:6]: Interesting that using a newer Visual Studio Version changes the project GUID. It does not seem like it affects anything really so I guess we can keep the changes.

**Review by lulzsun — APPROVED** [review]
Tested it, looks and works as expected. Thanks! Ready to merge.

I do want to mention that this pull request did contain some unrelated changes to files to the following files:
- Program.cs
- RePlays.csproj
- RePlays.sln

While these changes were minor and/or most likely auto-generated, I recommend to be mindful of committing them in a pull request if they aren't related to the main changes of a PR just so we can keep commit history relevant.

Since they don't affect the project's functionality, I'm okay with overlooking them this time. Thanks again for the pull request.

**Comment by sshcrack** [comment]
Alright, will be careful not to commit unrelated changes. Thanks for reviewing!

**Pull request merged**